### PR TITLE
Clean up debug webhooks guide

### PR DESCRIPTION
**Pull request by alexisintech**

### CODE REVIEW

File: docs/integrations/webhooks/debug-your-webhooks.mdx [hunk 1]
```diff
@@ -58,53 +52,58 @@ export const config = {
...
-While your application is running, try this from the command line.
-
-```bash
-curl -H 'Content-Type: application/json' \
-     -X POST http://localhost:3000/api/webhooks/test
-```
-
-If you see the `{"message":"The route is working"}` then the basic Route Handler is working, and ready to build on.
-
+If you are having trouble with your webhook, you can create a basic response and then test it locally.
```
Comment (S3Prototype) [inline docs/integrations/webhooks/debug-your-webhooks.mdx:55]:
```suggestion
If you'e having trouble with your webhook, you can create a basic response and then test it locally.
```

File: docs/integrations/webhooks/debug-your-webhooks.mdx [hunk 2]
```diff
@@ -114,10 +113,10 @@ Diagnosing a problem in this part of the webhook can be challenging. Your best b
 
 The results of these will appear in the command line where you are running your application.
 
-Checking the values and the progress of the webhook code will allow you to narrow down where the code is failing. The will often return null or errors where they should be returning values.
+Checking the values and the progress of the webhook code will allow you to narrow down where the code is failing. The will often return `null` or errors where they should be returning values.
```
Comment (S3Prototype) [inline docs/integrations/webhooks/debug-your-webhooks.mdx:116]:
```suggestion
Checking the values and the progress of the webhook code will allow you to narrow down where the code is failing. They will often return `null` or errors where they should be returning values.
```

File: docs/integrations/webhooks/debug-your-webhooks.mdx [hunk 3]
```diff
@@ -114,10 +113,10 @@ Diagnosing a problem in this part of the webhook can be challenging. Your best b
...
 
 ## Check your logic
 
 Once you have verified the webhook, you will now be writing your own code to handle the values from the webhook. This could range from saving data to a databse, or integrating with another system, to updating users or sending emails or SMS.
 
-If the webhook is verified and you are seeing a 500 status code or your webhook is not behaving as expected, remember that you can use `console.log` to help diagnose what the problem is. Console logs and error will be displayed on your command line, allowing you to see what's happening and address the bugs.
+If the webhook is verified and you are seeing a `500` status code or your webhook is not behaving as expected, remember that you can use `console.log` to help diagnose what the problem is. Console logs and error will be displayed on your command line, allowing you to see what's happening and address the bugs.
```
Comment (S3Prototype) [inline docs/integrations/webhooks/debug-your-webhooks.mdx:122]:
```suggestion
If the webhook is verified and you're seeing a `500` status code or your webhook is not behaving as expected, remember that you can use `console.log` to help diagnose what the problem is. Console logs and errors will be displayed on your command line, allowing you to see what's happening and address the bugs.
```

**Review by S3Prototype — COMMENTED** [review]
Will test the code in a bit

**Comment by alexisintech** [comment]
> Will test the code in a bit



> Will test the code in a bit

No new code was added, and it was tested by both Roy and I when it was originally added - so you don't have to if you don't feel the need to !

**Review by S3Prototype — APPROVED** [review]

**Pull request merged**